st2common/st2common/models/db/liveaction.py

```diff
@@ -75,6 +75,7 @@ class LiveActionDB(stormbase.StormFoundationDB):
         default={},
         help_text='Information about the runner which executed this live action (hostname, pid).')
     notify = me.EmbeddedDocumentField(NotificationSchema)
+    log = me.ListField(field=me.DictField())
```

Comment:
All that said, just add log to execution model. You don't need this in liveaction.